src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,13 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
```

Comment:
```suggestion
export const CommentInfo = ({ comment: { name, email, body } }) => (
```
Consider using destructuring here